Jill_code_review/client/index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html ng-app='app'> 
+	<head>
+		<title></title>
+ <!-- THESE TWO VERSIONS BELOW MUST MATCH (good thing bower will catch that for us, but be aware if you are using CDN's)!!! -->
+		<script src="angular/angular.js"></script>
+		<script src="angular-route/angular-route.js"></script>
+		<script>
+			//  inject the ngRoute dependency in the module.
+			var app = angular.module('app', ['ngRoute']);
+			//  use the config method to set up routing:
+			app.config(function ($routeProvider) {
+				$routeProvider
+					.when('/players',{
+							templateUrl: 'partials/players.html',
+					})
+					.when('/teams',{
+							templateUrl: 'partials/teams.html',
+					})
+					.when('/associations',{
+							templateUrl: 'partials/associations.html',
+					})
+					.otherwise({
+						redirectTo: '/players'
+					});
+			});
+
+			app.factory('playerFactory', function(){
+				var players = [
+						{name: 'Speros'},
+						{name: 'Jimmy'},
+						{name: 'Jay'}
+					];
+					// factory always contains methods
+					var factory = {};
```

Comment:
We want to make sure we follow proper indentation rules even if JS isn't reliant on them. `var players` and `var factory` should be indented to the same distance.